.github/ISSUE_TEMPLATE/package_issue.yml

```diff
@@ -0,0 +1,76 @@
+name: 'Package Issue'
+description: Report issues and unexpected behaviour of packages with WinGet Client
+title: "[Package Issue]: Package.Identifer"
+labels: ["Issue-Bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please create issues related to the winget.exe client [here](https://github.com/microsoft/winget-cli/issues/new/choose)
+  - type: checkboxes
+    attributes:
+      label: Please confirm these before moving forward
+      description: Please confirm the following before posting your issue.
+      options:
+        - label: I have search for my issue and not found a work-in-progress/duplicate/resolved issue.
+          required: true
+        - label: I have not been informed if the issue is resolved in a preview version of the winget client.
```

Comment:
Maybe we also want a link pointing to the latest release notes of the preview versions?

Reply:
Releases only point to "stable" rather than "pre-releases" otherwise, we have to directly link to preview releases :( Unless you know something about GitHub that I don't 😊

Reply:
I was just thinking that if you look at https://github.com/microsoft/winget-cli/releases the ones marked as pre-release aren't used for things like SandboxTest.ps1 and they do contain bug fixes that not everyone may be aware of;  [This Issue](https://github.com/microsoft/winget-pkgs/issues/47431) is a good example

Reply:
Let's see how this goes. We can iterate to improve it if we're not getting good results.